models/DataObject/ClassDefinition/Helper/VarExport.php

```diff
@@ -47,9 +47,6 @@ public static function __set_state(array $data): static
         return $obj;
     }
 
-    /**
```

Comment:
why is `$this` removed here? also in other places?

Reply:
Because the method is also defined in the parent class. The php docs are inherit.